<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>621</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Book</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selamat datang admin</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminImage">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Monotype Corsiva</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminName">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>571</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Book</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
